src/breadcrumbs/breadcrumbs.scss

```
@@ -50,9 +50,25 @@
     }
   }
 
+  > span:not([class]) {
+    padding: 0 $iui-s;
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+  }
+
   .iui-button {
-    @include iui-button-borderless;
-    border: none;
+    &,
+    &.iui-default {
+      @include iui-button-borderless;
+      background-color: transparent;
+      border: none;
+
+      &:focus-visible {
+        outline-offset: -1px;
+        outline-width: 1px;
+      }
+    }
```

Are you sure about all these overrides? I think it's just a specificity issue, which could be worked around by duplicating the class.

```suggestion
    &#{&} {
      @include iui-button-borderless;
      border: none;
    }
```

---

I believe this has been solved with [your suggestion here](https://github.com/iTwin/iTwinUI/pull/568#discussion_r863943914).